src/components/TextField/TextField.tsx

```diff
@@ -1,30 +1,18 @@
 import React from 'react';
 import { TextField as TextFieldMUI, TextFieldProps } from '@mui/material';
 
-export const TextField = ({
-  sx,
-  placeholder,
-  id = 'outlined-basic',
-  label = '',
-  variant = 'outlined',
-  disabled = false,
-  color = 'primary',
-  error = false,
-  helperText = '',
-  ...props
-}: TextFieldProps) => {
+export const TextField = ({ ...props }: TextFieldProps) => {
+  const { variant = 'outlined' } = props;
   return (
     <TextFieldMUI
       fullWidth
-      sx={sx}
-      label={label}
+      sx={{
```

Comment:
I wonder if it would be good idea to add these styling straight to theme, as if
```
components: {
      MuiTextField: {
        styleOverrides: {
          root: {
            padding: '1.5rem',
            paddingBottom: '0'
          }
        }
      },
}
```
. I was thinking same about Button `sx={{borderRadius: '8px'}}` what do you think?